### PR TITLE
Added auth param to run.sh and auth generator to setup.sh

### PR DESCRIPTION
**Pull request by ffigueroa**
## Short Description:
Added auth param to run.sh
Added auth generator to setup.sh
Added auth generator in install
Added auth.json in backup
Added .DS_Store to .gitignore
Modified config generator with options.

## Fixes/Resolves/Closes (please use correct syntax):
- Fixes https://github.com/PokemonGoF/PokemonGo-Bot/issues/4823
- No auth argument specified using run.sh

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -23,8 +23,8 @@ echo "There's no "$filename" file. Please use ./setup.sh -c to creat one."
 fi
 while true
 do
-python pokecli.py -cf $filename
-echo `date`" Pokebot "$*" Stopped." 
+python pokecli.py -af ./configs/auth.json -cf $filename
```
Comment (solderzzc) [inline run.sh]: This should be a param from commend too. How do you think about

Comment (ffigueroa) [inline run.sh]: and now? :)

**Comment by ffigueroa** [comment]
👍

**Comment by mjmadsen** [comment]
Thanks for the PR

**Pull request merged**